From c22cd52b1e611846ba9a8980fa6afc5d7c03e8e0 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Date: Mon, 14 Dec 2020 15:38:30 +0200
Subject: [PATCH 28/78] dts: s32g274: introduce 'fsl,s32g274a-siul2-gpio'

'fsl,s32g274a-siul2-gpio' is used for s32g274a to restrict opad ranges

Issue: ALB-6155
Signed-off-by: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi b/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi
index 28c81fcd62b7..bcdc037ae659 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g274a.dtsi
@@ -165,7 +165,7 @@
 			status = "okay";
 		};
 		gpio0: siul2-gpio0 {
-			compatible = "fsl,s32gen1-siul2-gpio";
+			compatible = "fsl,s32g274a-siul2-gpio";
 			gpio-controller;
 			#gpio-cells = <2>;
 				/* GPIO 0-101 */
@@ -223,7 +223,7 @@
 		 * assume this for many things.
 		 */
 		gpio1: siul2-gpio1 {
-			compatible = "fsl,s32gen1-siul2-gpio";
+			compatible = "fsl,s32g274a-siul2-gpio";
 			gpio-controller;
 			#gpio-cells = <2>;
 				/* GPIO 112-190 */
-- 
2.25.1

